p4broker/config/p4broker.conf

```diff
@@ -0,0 +1,204 @@
+#-------------------------------------------------------------------------------
+# Sample p4broker configuration file
+#
+# The following paragraphs illustrate the configuration of the Perforce
+# Broker by example. Change the values and configuration to suit your
+# environment and requirements.
+#
+# Note that by default this broker will simply allow all commands to pass
+# through, and be executed against the target server. Uncomment and edit
+# the examples below to get started.
```

Comment:
Typically comments go into the `default.toml` so when you run `hab pkg config` you see them in the output.